src/test/java/persistence/sql/ddl/DataDefinitionLanguageGeneratorTest.java

```diff
@@ -0,0 +1,183 @@
+    private final DataDefinitionLanguageGenerator dataDefinitionLanguageGenerator = new DataDefinitionLanguageGenerator(
+        getTableNameFromClass, getFieldFromClass
+    );
+
+
+    @Test
+    @DisplayName("@Entity 가 없으면 CannotCreateTableException 이 발생한다.")
+    void TestWithNoEntity() {
+        assertThrows(
+            CannotCreateTableException.class,
+            () -> dataDefinitionLanguageGenerator.generateTableCreatorWithClass(NoEntityClass.class));
+    }
+
+    @Test
+    @DisplayName("id가 여러개면 FieldShouldHaveOnlyOnePrimaryKeyException 이 발생한다")
+    void TestWithMultipleId() {
+        assertThrows(
+            FieldShouldHaveOnlyOnePrimaryKeyException.class,
+            () -> dataDefinitionLanguageGenerator.generateTableCreatorWithClass(MultipleIdClass.class)
+        );
+    }
+
+    @Test
+    @DisplayName("@Table이 없다면 TableName 은 클래스 이름이다.")
+    void TestClassNameWithoutTable() {
+        TableCreator tableCreator = dataDefinitionLanguageGenerator.generateTableCreatorWithClass(
+            ClassWithoutTable.class);
+        assertThat(tableCreator.getTableName().getName()).isEqualTo("ClassWithoutTable");
+    }
+
+    @Test
+    @DisplayName("@Table이 있어도 name이 없다면 TableName 은 클래스 이름이다.")
+    void TestClassNameWithTableButNoName() {
+        TableCreator tableCreator = dataDefinitionLanguageGenerator.generateTableCreatorWithClass(
+            ClassWithTableButEmptyName.class);
+        assertThat(tableCreator.getTableName().getName()).isEqualTo("ClassWithTableButEmptyName");
+    }
+
+    @Test
+    @DisplayName("@Table이 있고 name이 있다면 TableName 은 지정한 이름이다.")
+    void TestClassNameWithTableAndName() {
+        TableCreator tableCreator = dataDefinitionLanguageGenerator.generateTableCreatorWithClass(
+            ClassWithTableAndName.class);
+        assertAll(
+            () -> assertThat(tableCreator.getTableName().getName()).isNotEqualTo("ClassWithTableAndName"),
+            () -> assertThat(tableCreator.getTableName().getName()).isEqualTo("testClass")
+        );
+    }
+
+    @Test
+    @DisplayName("@Column 이 지정되지 않았다면 클래스 필드명이 테이블 필드명이 된다")
+    void testFieldNameWithoutColumn() {
+        TableCreator tableCreator = dataDefinitionLanguageGenerator.generateTableCreatorWithClass(
+            ColumnTestClass.class);
+
+        List<DatabaseField> databaseFields = tableCreator.getFields().getDatabaseFields();
+        assertThat(databaseFields.stream().filter(it -> "noColumn".equals(it.getName())).count()).isEqualTo(1L);
+    }
+
+    @Test
+    @DisplayName("@Column 이 있어도 name이 빈 값이면 클래스 필드명이 테이블 필드명이 된다")
+    void testFieldNameWithColumnButNoName() {
+        TableCreator tableCreator = dataDefinitionLanguageGenerator.generateTableCreatorWithClass(
+            ColumnTestClass.class);
+
+        List<DatabaseField> databaseFields = tableCreator.getFields().getDatabaseFields();
+        assertThat(databaseFields.stream().filter(it -> "columnWithoutName".equals(it.getName())).count()).isEqualTo(
+            1L);
```

Comment:
```suggestion
        assertThat(databaseFields)
                .extracting(DatabaseField::getName)
                .contains("columnWithoutName");
```

가독성을 위한다면 이렇게 해볼수도 있어요! 🤔

Reply:
수정했습니다